lib/helpers/is-interactive-element.js

```diff
@@ -19,18 +19,27 @@ const INTERACTIVE_TAG_NAMES = [
 const ARIA_WIDGET_ROLES = [
```

Comment:
Would you mind updating the spec link just above this line to point to the current URL?